blockmanager.go

```diff
@@ -395,15 +395,23 @@ func (b *blockManager) cfHandler() {
 		b.wg.Done()
 	}()
 
-	log.Infof("Waiting for block headers to sync, then will start " +
-		"cfheaders sync...")
+	b.newFilterHeadersMtx.RLock()
+	filterHeaderTip := b.filterHeaderTip
+	filterHeaderTipHash := b.filterHeaderTipHash
```

Comment:
I don't think these local variables are actually needed as this is the only goroutine that can mutate these variables.

Reply:
Removed.